src/git/README.md

```diff
@@ -120,6 +120,19 @@ Add this to your `claude_desktop_config.json`:
 ```
 </details>
 
+<details>
+<summary>Using docker</summary>
+
+```json
+"mcpServers": {
+  "git": {
+    "command": "docker",
+    "args": ["run", "--rm", "-i", "type=bind,src=/Users/slim,dst=/projects", "mcp/git"]
```

Comment:
Is the username here deliberate? I think it's easy to miss that someoen needs to adjust the username. We should make a little note.

Reply:
fixed